### PR TITLE
Recipe buffs for opticals

### PR DESCRIPTION
**Pull request by TheEpicGamer274**
Some community members asked for a buff for some recipes along with an alternate recipe to reduce glowstone cost, and time cost from the memory 3.5 used in opticals.

### CODE REVIEW

File: src/main/java/com/dreammaster/gthandler/recipes/SpaceAssemblerRecipes.java [hunk 1]
```diff
@@ -100,6 +105,20 @@ public void run() {
...
             }
+
+            // Memory Tier 3.5
+            IG_RecipeAdder.addSpaceAssemblerRecipe(
+                    new ItemStack[] { ItemList.Circuit_Board_Elite.get(4), ItemList.Circuit_Chip_Ram.get(64),
+                            ItemList.Circuit_Chip_SoC.get(64), ItemList.Circuit_Chip_NAND.get(64),
+                            GT_OreDictUnificator.get(OrePrefixes.circuit, Materials.SuperconductorUHV, 1L),
+                            GT_OreDictUnificator.get(OrePrefixes.foil, Materials.VanadiumGallium, 64L) },
+                    new FluidStack[] { new FluidStack(solderIndalloy, 1152) },
+                    GT_ModHandler.getModItem(OpenComputers.ID, "item", 64L, 39),
```
Comment (chochem) [inline src/main/java/com/dreammaster/gthandler/recipes/SpaceAssemblerRecipes.java:116]: needs OC, put it in the OpenComputers.isModLoaded() bracket

File: src/main/java/com/dreammaster/gthandler/recipes/AssemblerRecipes.java [hunk 2]
```diff
@@ -3189,6 +3191,15 @@ public void run() {
                 100,
                 480);
 
+        // Irradiant Uranium
```
[the remark this answers is not in the document]
Comment (TheEpicGamer274) [reply src/main/java/com/dreammaster/gthandler/recipes/AssemblerRecipes.java:3194]: There should've been one before they moved all the recipes

[the remark this answers is not in the document]
Comment (chochem) [reply src/main/java/com/dreammaster/gthandler/recipes/AssemblerRecipes.java:3194]: there is one elsewhere in the file it seems

[the remark this answers is not in the document]
Comment (TheEpicGamer274) [reply src/main/java/com/dreammaster/gthandler/recipes/AssemblerRecipes.java:3194]: it doesn't look like it tbh

**Pull request merged**